DigitalLearningSolutions.Data.Tests/Services/SectionContentServiceTests.cs

```diff
@@ -382,6 +402,62 @@ public void Get_section_content_returns_null_if_customisation_is_inactive()
             result.Should().BeNull();
         }
 
+        [Test]
+        public void Get_section_content_should_not_use_archived_tutorials_in_scores()
+        {
+            // Given
+            const int customisationId = 22416;
+            const int candidateId = 118178;
+            const int sectionId = 1955;
+
+            // When
+            var result = sectionContentService.GetSectionContent(customisationId, candidateId, sectionId);
+
+            // Then
+            result.Should().NotBeNull();
+            result!.SectionScore.Should().Be(0);
+            result!.MaxSectionScore.Should().Be(2); // Not 3 as uspReturnSectionsForCandCust_V2 returns because
+                                                    // it counts archived tutorial 9366
+            result!.DiagnosticAttempts.Should().Be(1);
```

Comment:
Is this relevant to the test? 🙂

Reply:
Which bit? The comment isn't strictly necessary but I thought it was helpful to precisely locate the discrepancy for whoever uses this test in the future 🙂

Reply:
Sorry I meant checking the diagnostic attempts. It didn't seem relevant to what was being tested? I like the comment, I agree it's useful 🙂

Reply:
So the diagnostic attempts and the section score do both depend on whether archived tutorials are included or not (the section score being the sum of the tutorial scores, and the section diagnostic attempts being the maximum over the diagnostic attempt values for each tutorials in the section). As it happens both of these agree with the old stored procedure for this case, but they might not for every case. I think it's helpful to include because it checks the three values that including archived tutorials can effect, even if there only is a discrepancy in one value 🙂

Reply:
Alternatively I could temporarily edit the database to make all three of these values depend on whether we are including the archived tutorials or not? 🙂

Reply:
Ah I see. I think changing the db (in transaction scope) so they all depend on archived tutorials would make it a more useful test 🙂 I'd also be tempted to split it into two tests; one for the max score and one for the current score 🙂

Reply:
Yes I agree, added in 187795d 🙂